README.md

```diff
@@ -403,21 +390,27 @@ cd client; go test  -tags sqlite  .
 
 We use [Cypress](https://www.cypress.io) to run our e2e tests. You can run all tests using:
 
-```shell script
+<pre textrun="is-make-task">
 make test-e2e
-```
+</pre>
 
 If you intend developing e2e tests, run the following command for more details:
 
-```shell script
+<pre textrun="verify-executable-exists">
```

Comment:
This action checks that the file exists and is executable. It doesn't run it because I assume you verify that the file works as part of the test suite.